draft-ietf-quic-recovery.md

```diff
@@ -476,9 +476,10 @@ When ack-eliciting packets are in-flight in multiple packet number spaces,
 the timer MUST be set for the packet number space with the earliest timeout,
 except for ApplicationData, which MUST be ignored until the handshake
 completes; see Section 4.1.1 of {{QUIC-TLS}}.  Not arming the PTO for
-ApplicationData prioritizes completing the handshake and prevents the server
-from sending a 1-RTT packet on a PTO before before it has the keys to process
-a 1-RTT packet.
+ApplicationData prevents a client from retransmitting a 0-RTT packet on a PTO
+expiration before confirming that the server is able to decrypt 0-RTT packets,
+and prevents a server from sending a 1-RTT packet on a PTO expiration before it
+has the keys to process an acknowledgement.
```

Comment:
```suggestion
ApplicationData prevents a client from sending a packet on PTO where there is uncertainty about the ability of the server to acknowledge the packet. This can happen before a client confirms that the server is able to decrypt 0-RTT packets, and also at a server before it has the keys to process an acknowledgement.
```

Reply:
This isn't only about the server acknowledging packets, so I think this isn't quite correct.